BraveShared/BraveStrings.swift

```diff
@@ -157,7 +157,7 @@ extension Strings {
         public static let notificationBody =
             NSLocalizedString("defaultBrowserCallout.notificationBody",
                               tableName: "BraveShared", bundle: .braveShared,
-                              value: "Optimized for iOS 14. Make Brave your default browser today.",
+                              value: "Optimized for iOS \(floor(Double(UIDevice.current.systemVersion) ?? 13)). Make Brave your default browser today.",
```

Comment:
It can't be done here because these translations are exported to a translation service,
here you have to use `%@` character for iOS version placeholder,

then in code you get this string like `String(format: notificationBody, your_number_logic)`

You can search for `findInPageFormat` in our code to see how other string with a placeholder is set up